sdk/containerregistry/azure-containerregistry/azure/containerregistry/_anonymous_exchange_client.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, endpoint, **kwargs):  # pylint: disable=missing-client-constr
             endpoint = "https://" + endpoint
         self._endpoint = endpoint
         self._client = ContainerRegistry(
-            credential=None,
+            credential="", # type: ignore
```

Comment:
Does "" mean something different than None to the service?

Reply:
"" here is equivalent to None in previous generation.
The parameter "credential" becomes `required` in this generation, and the expected type is `TokenCredential`, I have to pass empty string to avoid the None check when passing down in anonymous access.